### DISCUSSION TITLE
CameraComponent.ScreenToWorldPoint returned a Nan Vector3

### DISCUSSION THREAD

**Discussion by DJDQfff** [stride3d/stride#2586] [Q&A]
Why    `  var b = CameraComponent.ScreenToWorldPoint( new(0.1f, 0.5f, 0.1f));`return a vector3 consisted of NaN.
Is my parameter wrong?

**Comment by Eideren** [comment]
This comes from the community toolkit, so might be best to ask over there next time. Is the camera you're using orthographic ? Maybe you called this method before the camera matrix could be initialized or is actually drawn ?
